common/lib/xmodule/xmodule/js/spec/sequence/display_spec.js

```diff
@@ -0,0 +1,104 @@
+/* globals Sequence */
+(function() {
+    'use strict';
+
+    describe('Sequence', function() {
+        var keydownHandler,
+            keys = {
+                ENTER: 13,
+                LEFT: 37,
+                UP: 38,
+                RIGHT: 39,
+                DOWN: 40
+            };
+        beforeEach(function() {
+            loadFixtures('sequence.html');
+            window.XBlock = jasmine.createSpyObj('XBlock', ['initializeBlocks']);
+        });
+
+        afterEach(function() {
+            delete window.XBlock;
```

Comment:
There might be a better way to do this - after the last test has run, window.XBlock won't be defined any more, which could cause problems for other describe blocks or specs that follow this one.